airflow/operators/subdag.py

```diff
@@ -78,6 +86,13 @@ def __init__(
         self._validate_dag(kwargs)
         self._validate_pool(session)
 
+        warnings.warn(
+            """This class is deprecated.
+            Please use `airflow.utils.task_group.TaskGroup`.""",
```

Comment:
I'm wondering if we have any subdag to tg migration guide / howto. If yes, then maybe it would be nice to point to it in this warning?

Reply:
We have an open task for it:
https://github.com/apache/airflow/issues/12298